Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -1641,6 +1656,60 @@ def checkpoint_show_task_command(client: Client, task_id: str) -> CommandResults
     return command_results
 
 
+def checkpoint_add_objects_batch_command(client: Client, object_type: str, ipaddress, name):
+    printable_result = {}
+    readable_output = ''
+
+    ipaddress = argToList(ipaddress, ',')
+    name = argToList(name, ',')
+    add_list = []
+    for ip, n in zip(ipaddress, name):
+        tmp_dict = {'name': n, 'ip-address': ip}
+        add_list.append(tmp_dict)
+
+    result = current_result = client.add_objects_batch(object_type, add_list)
+
+    if result:
+        printable_result = {'task-id': result.get('task-id')}
+        readable_output = tableToMarkdown('CheckPoint data for add-objects-batch command:',
+                                          printable_result)
+
+    command_results = CommandResults(
+        outputs_prefix='CheckPoint.add_objects_batch',
+        outputs_key_field='uid',
+        readable_output=readable_output,
+        outputs=printable_result,
+        raw_response=result
+    )
+    return command_results
+
+
+def checkpoint_delete_objects_batch_command(client: Client, object_type: str, name):
+    printable_result = {}
+    readable_output = ''
+
+    name = argToList(name)
+    del_list = []
+    for n in name:
+        tmp_dict = {'name': n}
+        del_list.append(tmp_dict)
```

Comment:
And obviously use the `objects_to_delete` name on other locations

```suggestion
    objects_to_delete = [{'name':object_name} for object_name in object_names]
```